lib/ansible_tower_client/collection_methods.rb

```diff
@@ -13,7 +13,7 @@ def all
     end
 
     def find(id)
-      body = JSON.parse(Api.get("#{self.endpoint}/#{id}/").body)
+      body = JSON.parse(Api.get("#{endpoint}/#{id}/").body)
       raise NoMethodError if body['id'].nil?
```

Comment:
Sorry, I should have mentioned this on the other PR... I think `NoMethodError` is strange to use here.  Maybe we need a `ResourceNotFoundError` or something like that.  Does Faraday have anything like this?